apps/functions-subscription/src/adapters/azure/cosmosdb/activation-job.ts

```diff
@@ -0,0 +1,19 @@
+import { pipe } from 'fp-ts/lib/function';
+import * as E from 'fp-ts/lib/Either';
+import * as TE from 'fp-ts/lib/TaskEither';
+import { Database } from '@azure/cosmos';
+import { ActivationJobWriter } from '../../../domain/activation-job';
+import { cosmosErrorToDomainError } from './errors';
+
+export const makeActivationJobCosmosContainer = (
+  db: Database,
+): ActivationJobWriter => {
+  const container = db.container('activations');
```

Comment:
**note**: At the moment it is hardcoded but we can refactor and require as parameter of `makeActivationJobCosmosContainer`

Reply:
I'd do in a separate PR